src/controllers/auth.js

```diff
@@ -1,2 +1,37 @@
 const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
 const UserModel = require('../models/user');
+
+const loginUsers = async (req, res) => {
+  try {
+    const { email, password } = req.body;
+    const databaseUser = await UserModel.findOne({ email });
+
+    if (!databaseUser) {
+      return res.status(401).json({ message: 'Wrong username or password!' });
+    }
+    const validPassword = await bcrypt.compare(
+      password,
+      databaseUser.passwordHash
+    );
+    if (!validPassword) {
+      return res.status(401).json({ message: 'Wrong username or password!' });
+    }
+    // eslint-disable-next-line no-underscore-dangle
+    const payload = { userId: databaseUser._id };
+    const token = jwt.sign(payload, process.env.SECRET_KEY, {
+      expiresIn: '14d',
```

Comment:
can we not use magic strings please, lets put this in a variable in our const file and import it from there